experiments/compositions/composition/expanders/expand/main.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"flag"
+	"log"
+	"time"
+
+	pb "google.com/composition/proto"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+var (
+	addr = flag.String("addr", "[::]:50051", "the address to connect to")
+	cmd  = flag.String("command", "validate", "validate|expand")
```

Comment:
Why do we have expand here?

Reply:
should be evaluate. fixed it. This file is a simple grcp cli for test. will expand on the cli to get params from input files etc.